src/init.cpp

```diff
@@ -999,6 +1000,7 @@ bool AppInit2(boost::thread_group& threadGroup, CScheduler& scheduler)
                                        mapArgs["-maxtxfee"], ::minRelayTxFee.ToString()));
         }
     }
+    nBlockExpiry = GetArg("-txblockexpiry", DEFAULT_TX_EXPIRY_DELTA);
```

Comment:
Is `-txblockexpiry` clear enough on its own, or should it indicate somehow that it is a relative value?

Reply:
Thought @ebfull 's suggestion of changing constant's name to include `DELTA` helps. Can add a comment above